plugins/main/public/components/common/tables/table-wz-api.tsx

```diff
@@ -77,7 +79,7 @@ export function TableWzAPI({
   const [totalItems, setTotalItems] = useState(0);
   const [filters, setFilters] = useState({});
   const [isLoading, setIsLoading] = useState(false);
-
+  const [sort, setSort] = useState({ field: 'file', direction: 'asc' });
```

Comment:
**nitpick**: this component is used for multiple data types and the `file` field could not be present.